components/eppp_link/CMakeLists.txt

```diff
@@ -1,3 +1,16 @@
-idf_component_register(SRCS eppp_link.c eppp_sdio_slave.c eppp_sdio_host.c
+if(CONFIG_EPPP_LINK_DEVICE_ETH)
+    set(transport_src eppp_eth.c)
+endif()
+
+if(CONFIG_EPPP_LINK_DEVICE_SDIO)
+    set(transport_src eppp_sdio_slave.c eppp_sdio_host.c)
+endif()
+
+idf_component_register(SRCS eppp_link.c ${transport_src}
                     INCLUDE_DIRS "include"
-                    PRIV_REQUIRES esp_netif esp_driver_spi esp_driver_gpio esp_timer driver)
+                    PRIV_REQUIRES esp_netif esp_driver_spi esp_driver_gpio esp_timer driver esp_eth)
```

Comment:
I'm not sure if `driver` is not going to be obsolete soon. On the other hand, if you just removed it, it wouldn't work with older IDF version. Please consider handle it as https://github.com/espressif/esp-eth-drivers/blob/master/enc28j60/CMakeLists.txt#L3

Reply:
Thanks for the pointers. Have updated!